File: jdroid-android-about/src/main/java/com/jdroid/android/about/LibrariesActivity.java
package com.jdroid.android.about;

import android.support.v4.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class LibrariesActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return AboutAppModule.get().getAboutContext().getLibrariesFragmentClass();
	}
}

File: jdroid-android-about/src/main/java/com/jdroid/android/about/LibrariesActivity.kt
package com.jdroid.android.about

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class LibrariesActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return AboutAppModule.get().aboutContext.getLibrariesFragmentClass()
    }
}
